internal/controller/gcpupgrade/inputs.go

```diff
@@ -0,0 +1,257 @@
+package gcpupgrade
+
+import (
+	"bytes"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+
+	"flightcrew.io/cli/internal/controller"
+	"flightcrew.io/cli/internal/debug"
+	"flightcrew.io/cli/internal/gcp"
+	"flightcrew.io/cli/internal/view/wrapinput"
+)
+
+var (
+	initialInputKeys = []string{
+		keyProject,
+		keyVirtualMachine,
+		keyZone,
+		keyTowerVersion,
+	}
+)
+
+type InputsController struct {
+	inputs    map[string]*wrapinput.Model
+	args      map[string]string
+	inputKeys []string
+}
+
+func NewInputsController(params Params) *InputsController {
+	ctl := &InputsController{
+		inputKeys: initialInputKeys,
+		inputs:    make(map[string]*wrapinput.Model),
+		args:      params.args,
+	}
+
+	if !contains(ctl.args, keyVirtualMachine) {
+		ctl.args[keyVirtualMachine] = "flightcrew-control-tower"
+	}
+	if !contains(ctl.args, keyZone) {
+		ctl.args[keyZone] = "us-central1-c"
+	}
+	if !contains(ctl.args, keyTowerVersion) {
+		ctl.args[keyTowerVersion] = "stable"
+	}
+
+	ctl.args[keyImagePath] = gcp.ImagePath
+
+	for _, key := range allKeys {
+		var input wrapinput.Model
+		maybeSetValue := func(key string) {
+			if val, ok := ctl.args[key]; ok {
+				input.SetValue(val)
+			}
+		}
+
+		switch key {
+		case keyProject:
```

Comment:
Any way to reuse these from the other command?

Reply:
Moved stuff around.